docs/admin.md

```diff
@@ -0,0 +1,96 @@
+# Administrator Guide
+
+## Backstage Operator configuration
+
+### Context
+
+As it is described in Design doc (TODO), Backstage CR's desired state is defined using layered configuration approach, which means:
+- By default each newly created Backstage CR uses Operator scope Default Configuration
+- Which can be fully or partially overriden for particular CR instance using ConfigMap with the name pointed in BackstageCR.spec.RawConfig 
+- Which in turn can be customized by other BackstageCR.spec fields (see Backstage API doc)
+
+Cluster Administrator may want to customize Default Configuration due to internal preferences/limitations, for example:
+- Preferences/restrictions for Backstage and|or PostgreSQL images due to  Airgapped environment.
+- Existed Taints and tolerations policy, so Backstage Pods have to be configured with certain tolerations restrictions.
```

Comment:
you have capitalized Taints but not Tolerations. Intentional?

Reply:
no, it's typo